masonry/src/core/widget.rs

```diff
@@ -50,6 +50,12 @@ impl WidgetId {
     }
 }
 
+impl Default for WidgetId {
+    fn default() -> Self {
+        Self::next()
+    }
+}
+
```

Comment:
Big no to this. `Default` implementations that return different things if you call them twice are a huge code smell.